app.rb

```diff
@@ -0,0 +1,75 @@
+ENV["RACK_ENV"] ||= "development"
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative 'app/data_mapper_setup'
+
+
+class Chitter < Sinatra::Base
+
+  set :root, 'app'
+
+  enable :sessions
+  set :session_secret, 'super secret'
+
+  register Sinatra::Flash
+
+  use Rack::MethodOverride
+
+  get '/' do
+    erb :'sessions/new'
+  end
+
+  get '/users/new' do
+    erb :'users/new'
+  end
+
+  post '/users' do
+    user = User.create(
+      email: params[:email],
+      password: params[:password]
+    )
+    session[:user_id] = user.id
```

Comment:
Could the lines than set session user_id be refactored to use a helper function? eg:

```
def log_in_user(user_id)
  session[:user_id] = user_id
end
```

That way, the idea of how to log in a user is abstracted into one place.  This makes the mechanism of logging in a user easier to change.  It also makes it clearer what the code is doing.